src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -337,9 +338,6 @@ def build_cmd(self, env, host, **kwargs):
             hostfile = write_host_file(tst_host, daos_test_shared_dir, tst_ppn)
         mca_flags = ["btl self,tcp"]
 
-        if self.provider == "ofi+psm2":
-            mca_flags.append("pml ob1")
-
```

Comment:
I had that removed in the psm2 PR, it would be good if you keep it for now

Reply:
why? it should do a clean merge since we both removed the same